h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -164,6 +166,10 @@
   volatile int clusterPort = -1;
   volatile String flatfileContent = null;
 
+  private static Credentials make(String user) {
+    return Credentials.make(user, SecurityUtils.passwordGenerator(GEN_PASSWORD_LENGTH));
```

Comment:
Minor: This is related to an older PR, but you have a convenience method that only calls private constructor in exactly the same manner. Is there a reason for that ? Wouldn't it be better to just make the constructor public ? Or just make a convenience method that uses `SecurityUtils.passwordGenerator` inside ? Something like `Credentials.withGeneratedPassword` .... 

Just an observation. Not important.

Reply:
you are right, fixed